stackle-app/src/app/app.module.ts

```diff
@@ -22,6 +22,7 @@ import { SecureComponent } from './secure/secure.component';
 import { CommonFeedComponent } from './secure/common-feed/common-feed.component';
 import { ProfileComponent } from './secure/profile/profile.component';
 import { LoginComponent } from './public/login/login.component';
+import {PostComponent} from "./secure/shared/post/post.component";
```

Comment:
Check for lint issues. Add the spacing as necessary. Use single quotes instead of double quotes